### ISSUE TITLE
Import errors when building sphinx docs

### ISSUE THREAD

**Issue by chronitis** [xonsh/xonsh#1504]
`(debian/unstable amd64, python 3.5.2, sphinx 1.4.5)`

With 0.4.4, I get an `ImportError: _winapi` when trying to compile `docs/api/proc.rst`.

I'm not entirely certain why this is occuring since `_winapi` is listed under `:exclude-members:`, but my guess is `sphinx` still touches the object sufficiently to cause `LazyObject` to try and run the import, which isn't protected by a `ON_WINDOWS` guard.

Replacing (`xonsh/proc.py:30`)

    _winapi = LazyObject(lambda: importlib.import_module('_winapi'), globals(), '_winapi')

with

```
def _winapi():
    if ON_WINDOWS:
        import _winapi as w
    else:
        w = None
    return w
```

as is already done for `msvcrt` appears to fix this (but I haven't tested this on windows).

**Comment by scopatz** [comment]
HI @chronitis, thanks for reporting! - sorry for taking so long to respond.  I believe that this fixed on master. If you don't mind testing that and/or closing the issue that would be great!

**Comment by chronitis** [comment]
This was still present in 0.4.5, but I just checked master and it now works (presumably fixed by d096356d793ed33263db830ebac76ca4a4440c5d ).

**Comment by scopatz** [comment]
Yeah, just found it yesterday :) Thanks a ton for reporting!
